db/influx.go

```diff
@@ -391,8 +391,8 @@ func (i *influxDB) AppRelays(from, to time.Time) (map[string]api.RelayCounts, er
 }
 
 // startOfDay returns the time matching the start of the day of the input.
-//
 //	timezone/location is maintained.
+// TODO move this to utils-go
```

Comment:
OK to move to a separate package IMO if this is going to be used in multiple places, but hopefully to a small, focused package with a name indicating what it does, rather than `utils-go`

Reply:
We already have a `time` subpackage inside `utils-go`  for time related funcs so I was thinking to put it there.